chrome-extension/content.js

```diff
@@ -62,7 +62,7 @@ function closeSidebar() {
 }
 
 // Inject sidebar html content
-function createSidebar(content) {
+async function createSidebar(content) {
```

Comment:
Will look into this

Reply:
fixed i think?

Reply:
jc, why did adding async fix this? I was thinking a different thing you can try is moving `sidebar.appendChild(form);` to the end of this function. That way instead of the browser re-rendering every time you appendChild, it just does it one time when you append the form to the sidebar.

Reply:
oh the async didn't- it was there before too. What I thought what fixed it was moving `sidebar.appendChild(form)` to the beginning of the function. Before I had `sidebar.appendChild(form)` at the end of the function, and it was waiting for everything to load before showing up altogether, which is prob why there was a lag. Now there's no lag, but i didn't know it gets rerendered every time i appendChild. Is that bad?